newsroom/utils.py

```diff
@@ -504,15 +504,21 @@ def today(time, offset):
 
 
 def format_date(date, time, offset):
+    _today = today(time, offset)
     if date == "now/d":
         return today(time, offset)
     if date == "now/w":
-        _today = today(time, offset)
         monday = _today - timedelta(days=_today.weekday())
         return monday
     if date == "now/M":
         month = today(time, offset).replace(day=1)
         return month
+    if date == "now-24h/h":
+        return _today - timedelta(hours=24)
```

Comment:
I think here it should also set `.replace(minutes=0, seconds=0)` but not sure how it's actually used

Reply:
you're right about using `.replace(minute=0, second=0)` to ensure the datetime starts at the hour. However, since `_today` is already at `"00:00:00"`, it's unnecessary after subtracting `24` hours because the result will naturally be aligned to the hour.